composer.json

```diff
@@ -17,8 +17,8 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "container-interop/container-interop": "^1.1",
         "psr/http-message": "^1.0",
+        "psr/container": "^1.0",
```

Comment:
@xtreamwayz Can you re-target this PR at the develop branch, and add the `conflict` statement I've suggested above? If so, I'll give it a spin, and see how my various applications work with it.